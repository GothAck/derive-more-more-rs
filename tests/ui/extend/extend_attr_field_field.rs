use derive_more_more::Extend;

#[derive(Default, Extend)]
struct NamedMultiAttrStructField {
    #[extend(usize)]
    inner: Vec<usize>,
    #[extend(usize)]
    test: bool,
}

#[derive(Default, Extend)]
struct UnnamedMultiAttrStructField(#[extend(usize)] Vec<usize>, #[extend(usize)] bool);

fn main() {}
