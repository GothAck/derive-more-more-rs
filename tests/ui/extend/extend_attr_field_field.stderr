error: More than one field with `extend` attribute
 --> tests/ui/extend/extend_attr_field_field.rs:7:5
  |
7 |     #[extend(usize)]
  |     ^

error: More than one field with `extend` attribute
 --> tests/ui/extend/extend_attr_field_field.rs:5:5
  |
5 |     #[extend(usize)]
  |     ^

error: More than one field with `extend` attribute
  --> tests/ui/extend/extend_attr_field_field.rs:12:65
   |
12 | struct UnnamedMultiAttrStructField(#[extend(usize)] Vec<usize>, #[extend(usize)] bool);
   |                                                                 ^

error: More than one field with `extend` attribute
  --> tests/ui/extend/extend_attr_field_field.rs:12:36
   |
12 | struct UnnamedMultiAttrStructField(#[extend(usize)] Vec<usize>, #[extend(usize)] bool);
   |                                    ^
